database/scripts/schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE updates (
+    game_id         integer NOT NULL,
+    posted_by       char(100),
+    post_date       char(100),
+    title           char(100),
+    text            text,
+    img             text,
+    comment_count   integer
+);
```

Comment:
Everything looks good man. Except for those lines missing.

Reply:
Ok, I fixed them. Thanks. 
64393ba